lib/apparmortest.pm

```diff
@@ -632,6 +632,38 @@ sub adminer_database_delete {
     clear_console;
 }
 
+# Yast2 Apparmor set up
+sub yast2_apparmor_setup {
+    # Check Apparmor is active
+    systemctl("is-active apparmor");
+
+    # Turn to x11 and start "xterm"
+    select_console("x11");
+    x11_start_program("xterm");
+    become_root;
+}
+
+# Yast2 Apparmor: check apparmor is enabled
+sub yast2_apparmor_is_enabled {
+    type_string("yast2 apparmor &\n");
+    assert_screen("AppArmor-Configuration-Settings");
+    send_key "alt-l";
+    assert_screen("AppArmor-Settings-Enable-Apparmor");
+}
+
+# Yast2 Apparmor clean up
+sub yast2_apparmor_cleanup {
+    # Exit x11 and turn to console
+    send_key "alt-f4";
+    assert_screen("generic-desktop");
+    select_console("root-console");
```

Comment:
I just look into the lib/apparmortest.pm,  some of them are **select_console 'root-console';**  and  some are  **select_console ("root-console");** . Would it be possible to unify them under the same style in the file?

Reply:
BTW, I think all are using **select_console 'x11';**  in the file. Right?

Reply:
I chose `select_console 'x11'` before, but recently (for new code and in future) I prefer `select_console("x11")`. The reason is It is much more like c coding style. 'select_console' may likely have more than 1 inputs in future.
But there are exceptions, e.g., `send_key` only has one input so I prefer send_key "$key" as most of us did.

Reply:
> BTW, I think all are using **select_console 'x11';** in the file. Right?

No, old code uses `select_console "root-console";` in new code I would like to use `select_console("root-console");
To be honest I really hope somebody can make a detailed coding style role we can follow.

Reply:
Yes, I agree with you. I think it's the coding style issue, both are working in the case. It's up to you that if you'd like to change it. Or probably it can make an enhancement to unify them in the future. :)